vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events_database_table.cc

```diff
@@ -24,6 +27,29 @@ namespace {
 
 constexpr char kTableName[] = "text_embedding_html_events";
 
+std::vector<float> ConvertStringToVector(std::string string) {
```

Comment:
This seems like it should be defined at a higher level so other functions can use it